apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -115,6 +117,16 @@ export class PublishAction extends BaseRushAction {
       `the package is older than the current latest, so in publishing workflows for older releases, providing ` +
       `a tag is important. When hotfix changes are made, this parameter defaults to 'hotfix'.`
     });
+    this._npmAccessLevel = this.defineChoiceParameter({
+      alternatives: ['public', 'restricted'],
+      parameterLongName: '--set-access-level',
+      parameterShortName: undefined,
+      description:
+      `The access option to pass to npm publish. By default NPM will publish scoped packages with an access ` +
+      `level of 'restricted'. Scoped packages can be published with an access level of 'public' by specifying ` +
+      `that value for this flag with the initial publication. NPM always publishes unscoped packages with an ` +
```

Comment:
@octogonz do you have thoughts on the description text given the findings re: `npm publish --access`. Perhaps:

```suggestion
      `that value for this flag with the publish action. NPM always publishes unscoped packages with an ` +
```